bazel/print_dependencies.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# Quick-and-dirty python to fetch dependency information
+
+import imp
+import json
+import os.path
+import re
+
+locs = imp.load_source('replocs', 'bazel/repository_locations.bzl')
+recipes = imp.load_source('replocs', 'bazel/target_recipes.bzl')
+recipe_versions_file = 'ci/build_container/build_recipes/versions.sh'
+
+def parse_recipe_details(name, text):
+    info = {}
+
+    # Remove dashes
+    prefix = name.upper().translate(None, '-')
+    scans = {
+        prefix + '_GIT_REPO' : 'git-repo',
+        prefix + '_GIT_SHA' : 'git-sha',
+        prefix + '_FILE_SHA256' : 'file-sha256',
+        prefix + '_FILE_URL' : 'file-url',
+    }
+
+
+    for env_var, json_key in scans.items():
```

Comment:
This could also be done by moving the SHAs etc. to a Python file that is then used by the shell script. Possibly not worth it given the plans around switching to `external_cmake` though; build recipes will only have two entries..